### PR TITLE
feat(python-stack): Usage examples of the Google Earth Engine

### PR DESCRIPTION
**Pull request by FunkySayu**

### CODE REVIEW

File: computation/imagefetcher/examples/colored.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python2
+
+"""Example of image collection manipulation.
+
+In this example, we will see how to manipulate images collections. Image
```
Comment (ArchangelX360) [inline computation/imagefetcher/examples/colored.py:5]: Image collections or Images collections, you need to chose :D

File: computation/imagefetcher/examples/colored.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+# the whole earth, which is quite long to compute. Let's reduce this to a part
+# of the Colorado state, using a definition of the region.
+rectangle = ee.Geometry.Rectangle(-101, 39, -102, 40)
+
+# Get the Landsat 8 collection, filtered on a date chunk.
+raw_collection = (ee.ImageCollection('LANDSAT/LC8_L1T')
+        .filterDate("2000-01-01", "2015-06-1")
+        .filterBounds(rectangle))
+
+# Compile the collection as a mosaic.
```
Comment (ArchangelX360) [inline computation/imagefetcher/examples/colored.py]: Wrong comment here I guess, median() might not be a mosaic creation.

**Comment by FunkySayu** [comment]
All fix done. Waiting second review.

**Comment by ArchangelX360** [comment]
LGTM

**Pull request merged**